e2e/LongHaul/device/Parameters.cs

```diff
@@ -44,5 +50,33 @@ internal class Parameters
             Required = false,
             HelpText = "The protocol over which a transport (i.e., MQTT, AMQP) communicates.")]
         public IotHubClientTransportProtocol TransportProtocol { get; set; }
+
+        [Option(
+            'j',
+            "JsonSerializer",
+            Default = JsonSerializingLibrary.SystemTextJson,
+            Required = false,
+            HelpText = "Which JSON serialization library for the device app to use (i.e., SystemTextJson, NewtonsoftJson)")]
```

Comment:
same here, also this `i.e.,` has a comma afterward while the previous one doesn't. I think either work but we should be consistent in theory

Reply:
I'll fix the missing comma